Instagram/Views/PostCell.m

```diff
@@ -0,0 +1,22 @@
+#import "PostCell.h"
+
+@implementation PostCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+}
```

Comment:
Can delete. In Swift this would have the keyword `override` to make it more obvious that it's just overriding the super implementation. If the super implements it and you don't need to override then it's better to not do so.